src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -167,6 +168,21 @@ public StudentListPanel getStudentListPanel() {
         return studentListPanel;
     }
 
+
+    /**
+     * Displays reminder on start.
+     * @throws CommandException when error executing the command.
+     * @throws ParseException when the error meets an error.
+     */
+    public void showReminder() throws CommandException, ParseException {
+        try {
+            CommandResult commandResult = logic.execute(RemindCommand.COMMAND_WORD);
+            resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
+        } catch (CommandException | ParseException e) {
+            resultDisplay.setFeedbackToUser(e.getMessage());
```

Comment:
I am curious how it will display if there are more lines in the reminder

Reply:
It will be scrollable since the result display panel seems to be fixed size